doc/replicasets.rst

```diff
@@ -0,0 +1,226 @@
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add roles to replicaset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets add-roles ROLE_NAME... [flags]
+
+Flags:
+
+* ``--replicaset`` - name of replicaset
+* ``--vshard-group`` - vshard group (for ``vshard-storage`` replicasets)
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Remove roles from replicaset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets remove-roles ROLE_NAME... [flags]
+
+Flags:
+
+* ``--replicaset`` - name of replicaset
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Set weight of replicaset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets set-weight WEIGHT [flags]
+
+Flags:
+
+* ``--replicaset`` - name of replicaset
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Set replicaset failover priority
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets set-failover-priority INSTANCE_NAME... [flags]
+
+Flags:
+
+* ``--replicaset`` - name of replicaset
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Bootstrap vshard
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets bootstrap-vshard [flags]
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Expel instance(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    cartridge replicasets expel INSTANCE_NAME... [flags]
+
+-------------------------------------------------------------------------------
+Example
+-------------------------------------------------------------------------------
+
+The default application is used.
+It contains ``instances.yml`` file with instances configuration:
+
+.. code-block:: yaml
+
+    ---
+    myapp.router:
+    advertise_uri: localhost:3301
+    http_port: 8081
+
+    myapp.s1-master:
+    advertise_uri: localhost:3302
+    http_port: 8082
+
+    myapp.s1-replica:
+    advertise_uri: localhost:3303
+    http_port: 8083
+
+    # other instances are hidden in this example
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create two replicasets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Join instances:
+
+.. code-block:: bash
+
+    cartridge replicasets join --replicaset s-1 s1-master s1-replica
+
+        • Join instance(s) s1-master, s1-replica to replicaset s-1
+        • Instance(s) s1-master, s1-replica was successfully joined to replicaset s-1
```

Comment:
```suggestion
        • Instance(s) s1-master and s1-replica have been successfully joined to replicaset s-1
```

Reply:
The same as above, joined via `, `. But I will fix `replicaset` -> `replica set`.

Reply:
Fixed logs and tests.